2019/03.md

```diff
@@ -71,9 +71,9 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |  5m | [Normative: Remove implementation-defined `typeof` behavior](https://github.com/tc39/ecma262/pull/1441) | Daniel Ehrenberg |
     | 15m | [Normative: CreateDynamicFunction early concatenates bodyText](https://github.com/tc39/ecma262/pull/1479) | Leo Balter |
     | 15m | [Normative: Make Async-from-Sync iterator object inaccessible to ECMAScript code](https://github.com/tc39/ecma262/pull/1474) | Editors |
-    |  5m | [Normative: Remove implementation-defined typeof behavior](https://github.com/tc39/ecma262/pull/1441) | Daniel Ehrenberg / Editors |
```

Comment:
Why remove this one?

Reply:
It's a duplicate; not sure how it got there.